Aquality.WinAppDriver/src/Aquality.WinAppDriver/Elements/Actions/KeyboardActions.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Aquality.Selenium.Core.Applications;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Utilities;
+using Aquality.WinAppDriver.Actions;
+using Aquality.WinAppDriver.Elements.Interfaces;
+using System;
+
+namespace Aquality.WinAppDriver.Elements.Actions
+{
+    /// <summary>
+    /// Implements Keyboard actions for a specific element.
```

Comment:
is there a difference between Keyboard actions for the whole app? the keyboard is the same as for the whole app as for the element of the app. Can we discuss?

Reply:
discussed, everything is correct, difference is in the start point of actions. SeleniumWD has these methods overloaded with and without an element, so do we.